src/compiler/transformers/remove-static-meta-properties.ts

```diff
@@ -16,32 +18,33 @@ export const removeStaticMetaProperties = (classNode: ts.ClassDeclaration): ts.C
   return classNode.members.filter((classMember) => {
     if (retrieveTsModifiers(classMember)?.some((m) => m.kind === ts.SyntaxKind.StaticKeyword)) {
       const memberName = (classMember.name as any).escapedText;
-      if (REMOVE_STATIC_GETTERS.has(memberName)) {
+
+      if (readOnlyArrayHasStringMember(STATIC_GETTERS_TO_REMOVE, memberName)) {
         return false;
       }
     }
     return true;
   });
 };
 
-// TODO(STENCIL-856): Move these properties to constants for better type safety within the codebase
 /**
- * A list of static getter names that are specific to Stencil to exclude from a class's member list
+ * A list of the static getters to remove here, which is a subset of the total
+ * set of static getters used by Stencil during the compilation process.
  */
-const REMOVE_STATIC_GETTERS = new Set([
-  'is',
-  'properties',
-  'encapsulation',
+const STATIC_GETTERS_TO_REMOVE = [
   'elementRef',
+  'encapsulation',
   'events',
+  'is',
   'listeners',
   'methods',
-  'states',
   'originalStyleUrls',
-  'styleMode',
+  'properties',
+  'states',
   'style',
-  'styles',
+  'styleMode',
   'styleUrl',
-  'watchers',
   'styleUrls',
-]);
+  'styles',
+  'watchers',
```

Comment:
the diff looks bigger than it really is here because I alphabetized these while I was in here